bin/cli.js

```diff
@@ -53,6 +54,12 @@ program
   .command('start')
   .description('start the server')
   .action(withVapid(async (vapid) => {
+    const portInUse = new services.PortChecker(vapid.config.port).perform();
```

Comment:
missing an await here, should be: `await new services.PortChecker(`...

Reply:
@ohnoimdead Good catch, thank you.